bot/bot.go

```diff
@@ -5,24 +5,25 @@ import (
 
 	log "github.com/sirupsen/logrus"
 
+	"github.com/jaimeteb/chatto/channels"
 	"github.com/jaimeteb/chatto/clf"
-	cmn "github.com/jaimeteb/chatto/common"
-	"github.com/jaimeteb/chatto/ext"
+	"github.com/jaimeteb/chatto/extension"
 	"github.com/jaimeteb/chatto/fsm"
+	"github.com/jaimeteb/chatto/query"
 	"github.com/spf13/viper"
 )
 
 // Bot models a bot with a Classifier and an FSM
 type Bot struct {
 	Name       string
 	Machines   fsm.StoreFSM
-	Domain     fsm.Domain
+	DB         *fsm.DB
```

Comment:
`Domain` is a generic term and does not describe what it provides. I though maybe `DB` made sense here but that's debatable.

Reply:
I thought about Domain at first because [Rasa calls it that](https://rasa.com/docs/rasa/domain/). In the future, I plan to add an actual database to store the conversation history, so `DB` may be confusing. But until then, I'll stick with it.

Reply:
I did not know that. I will revert this back to `Domain`. Sorry about that I got carried away here.

Reply:
No need to apologize!